src/EditorFeatures/Core/Implementation/ExtractInterface/AbstractExtractInterfaceCommandHandler.cs

```diff
@@ -28,10 +28,8 @@ protected AbstractExtractInterfaceCommandHandler(IThreadingContext threadingCont
 
         public string DisplayName => EditorFeaturesResources.Extract_Interface;
 
-        public VSCommanding.CommandState GetCommandState(ExtractInterfaceCommandArgs args)
-        {
-            return IsAvailable(args.SubjectBuffer, out var _) ? VSCommanding.CommandState.Available : VSCommanding.CommandState.Unspecified;
-        }
+        public VSCommanding.CommandState GetCommandState(ExtractInterfaceCommandArgs args) =>
+            IsAvailable(args.SubjectBuffer, out var _) ? VSCommanding.CommandState.Available : VSCommanding.CommandState.Unspecified;
```

Comment:
same.

Reply:
note: you can do `out _` which i believe is a bit more idiomatic than `out var _`